src/app/dropdown/dropdown.component.css

```diff
@@ -11,6 +11,16 @@ li.dropdown-menu-box {
     color: #737373;
     text-decoration: none;
 }
+::-webkit-scrollbar { 
+    display: none; 
+}
+
```

Comment:
@sk9331657 Everything looks good, please remove the unwanted spaces. Keep the codebase clean.